src/test/java/org/springframework/data/hazelcast/repository/query/QueryIT.java

```diff
@@ -568,4 +568,62 @@ public void findByIdLike() {
 		assertThat("All years matched", expectedYears, hasSize(0));
 	}
 
+	@Test
+	public void findByFirstnameContains() {
+		List<Person> matches = this.personRepository.findByFirstnameContains("ll");
+
+		assertThat("Wallace, 2xWilliam, Russell", matches.size(), equalTo(4));
+		assertThat("Wallace, 2xWilliam, Russell", matches,
+				hasItems(allOf(hasProperty("firstname", equalTo("Wallace")), hasProperty("lastname", equalTo("Beery"))),
+						allOf(hasProperty("firstname", equalTo("William")), hasProperty("lastname", equalTo("Holden"))),
+						allOf(hasProperty("firstname", equalTo("William")), hasProperty("lastname", equalTo("Hurt"))),
+						allOf(hasProperty("firstname", equalTo("Russell")), hasProperty("lastname", equalTo("Crowe")))));
+	}
+
+	@Test
+	public void findByFirstnameContainsAndLastnameStartsWithAllIgnoreCase() {
+		List<Person> matches = this.personRepository.findByFirstnameContainsAndLastnameStartsWithAllIgnoreCase("En", "tR");
+
+		assertThat("2xSpencer", matches.size(), equalTo(2));
+		assertThat("2xSpencer", matches,
+				hasItems(allOf(hasProperty("firstname", equalTo("Spencer")), hasProperty("lastname", equalTo("Tracy")))));
+
+	}
+
+	@Test
+	public void countByIdAfter() {
+		Long count = this.personRepository.countByIdAfter("2000");
+		assertThat("Oscars began in 1928", count, equalTo(15L));
```

Comment:
`Oscars began in 1928` seems leftover because of copy/paste

Reply:
Yes, fixed that